### PR TITLE
chore(macros/JsSidebar): update Russian locale

### PR DESCRIPTION
**Pull request by Ansyr**
### Problem

wrong translation
related to: https://github.com/mdn/translated-content/pull/18396

### Solution

  fix translation
  related to: https://github.com/mdn/translated-content/pull/18396

---

### Before

image

### After

link title in aside change to: Конкурентная модель и цикл событий
link title in breadcrumbs change to: Конкурентная модель и цикл событий
---

### CODE REVIEW

**Comment by caugner** [comment]
@mdn/yari-content-ru Could you take a look at this one word change? 🙏

**Comment by saionaro** [comment]
@Ansyr @caugner hi! It has to be just "Цикл событий" - that's how the article is titled right now.
I don't have a power to do reviews in this repo :)

File: kumascript/macros/JsSidebar.ejs [hunk 1]
```diff
@@ -103,7 +103,7 @@ var text = mdn.localStringMap({
     'Inheritance': 'Наследование и цепочка прототипов',
     'Strict_mode': 'Строгий режим',
     'Memory_management': 'Управление памятью',
-    'Event_loop': 'Параллельная модель и цикл событий',
+    'Event_loop': 'Конкурентная модель и цикл событий',
```
Comment (saionaro) [inline kumascript/macros/JsSidebar.ejs:106]:
```suggestion
    'Event_loop': 'Цикл событий',
```

**Review by saionaro — CHANGES_REQUESTED** [review]
@Ansyr @caugner hi! It has to be just "Цикл событий" - that's how the article is titled right now.

**Review by caugner — APPROVED** [review]

**Pull request merged**